### PR TITLE
CAL-1300:Problem with special characters display when show task in ca…

### PR DESCRIPTION
**Pull request by bourasom**
…lendar

### CODE REVIEW

File: services/src/main/java/org/exoplatform/task/util/TaskUtil.java [hunk 1]
```diff
@@ -776,14 +776,37 @@ public static Event buildEvent(Event event, Task task) {
           event.setPriority(Event.PRIORITY_NONE);
       }   
     }    
-    event.setSummary(encoder.encode(task.getTitle()));
+    event.setSummary(encodeCalendarString(task.getTitle()));
```
Comment (thomasdelhomenie) [inline services/src/main/java/org/exoplatform/task/util/TaskUtil.java]: Did you try to remove completely the encoding ? I did a quick test (with PLF 5.0) , the special characters are already correctly encoded on calendar side, so no need to add extra encoding.

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

**Comment by thomasdelhomenie** [comment]
@bourasom looks good. I did some tests on PLF 4.4.4 as well and it works fine. Just one thing, when I try to open the event I always have the following message : "The event no longer exists or has been moved to another calendar.". Same for you ? Do you know if it is an expected behavior ?

**Comment by bourasom** [comment]
I reproduce the problem. The issue is related to Calendar's Type which seems to not be set:
org.exoplatform.calendar.service.impl.CalendarServiceImpl.getTypeOfCalendar(CalendarServiceImpl.java:417). This should be raised in another issue.
Thanks

**Comment by thomasdelhomenie** [comment]
ok. I validate this PR. I let you open the issue. Thanks!

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**